src/app/vault/view.component.ts

```diff
@@ -48,4 +55,33 @@ export class ViewComponent extends BaseViewComponent implements OnChanges {
         this.platformUtilsService.eventTrack('View Password History');
         this.onViewCipherPasswordHistory.emit(this.cipher);
     }
+
+    copy(value: string, typeI18nKey: string, aType: string) {
+        if (value == null) {
+            return;
+        }
+
+        this.platformUtilsService.eventTrack('Copied ' + aType);
+        const copyOptions = this.win != null ? { window: this.win } : null;
+        this.platformUtilsService.copyToClipboard(value, copyOptions);
+        this.platformUtilsService.showToast('info', null,
+            this.i18nService.t('valueCopied', this.i18nService.t(typeI18nKey)));
+        this.minimizeIfNeeded();
+
+        if (typeI18nKey === 'password') {
+            this.eventService.collect(EventType.Cipher_ClientToggledHiddenFieldVisible, this.cipherId);
+        } else if (typeI18nKey === 'securityCode') {
+            this.eventService.collect(EventType.Cipher_ClientCopiedCardCode, this.cipherId);
+        } else if (aType === 'H_Field') {
+            this.eventService.collect(EventType.Cipher_ClientCopiedHiddenField, this.cipherId);
+        }
```

Comment:
instead of re-implementing all of this, you can just call `super.copy(...)` and then do you minimize stuff after that.

Reply:
Implemented with 566e0c0. Also added minimizeOnCopyToClipboardKey constant to jslib of this repo, since it's desktop specific.